dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/io.openliberty.grpcClient-1.0.feature

```diff
@@ -25,6 +25,7 @@ Subsystem-Name: gRPC Client 1.0
 -bundles=\
   io.openliberty.grpc.1.0.internal.common, \
   io.openliberty.grpc.1.0.internal.client, \
+  io.openliberty.grpc.1.0.internal.monitor,\
```

Comment:
Same here ... my guess is that `io.openliberty.grpc.1.0.internal.monitor` should only be pulled in by the auto feature.